src/generated/cloud/aiplatform/v1/README.md

```diff
@@ -14,38 +14,38 @@ learning expertise and effort.
 
 The main types to work with this crate are the clients:
 
-- [DatasetService]
-- [DeploymentResourcePoolService]
-- [EndpointService]
-- [EvaluationService]
-- [FeatureOnlineStoreAdminService]
-- [FeatureOnlineStoreService]
-- [FeatureRegistryService]
-- [FeaturestoreOnlineServingService]
-- [FeaturestoreService]
-- [GenAiCacheService]
-- [GenAiTuningService]
-- [IndexEndpointService]
-- [IndexService]
-- [JobService]
-- [LlmUtilityService]
-- [MatchService]
-- [MetadataService]
-- [MigrationService]
-- [ModelGardenService]
-- [ModelService]
-- [NotebookService]
-- [PersistentResourceService]
-- [PipelineService]
-- [PredictionService]
-- [ReasoningEngineExecutionService]
-- [ReasoningEngineService]
-- [ScheduleService]
-- [SpecialistPoolService]
-- [TensorboardService]
-- [VertexRagDataService]
-- [VertexRagService]
-- [VizierService]
+- [DatasetService]: enabled by the `dataset_service` feature
```

Comment:
We could potentially link the feature flags page, but I am not sure the information is useful. e.g.

https://docs.rs/crate/tokio/latest/features#bytes

Maybe it could be made useful? #1901